### DISCUSSION TITLE
Platform/Host

### DISCUSSION THREAD

**Discussion by tayler6000** [eigentrustnetwork/ETN#1] [General]
Personally I was thinking we would make AE use Flask as an interface, it's super easy and well documented, however I'm open to ideas about this as well.  The main question is how do we want to host AE, and how are we going to store information?

In these early stages of development I'm going to start off using Flask and sqlite3, but I have no problems changing this if anyone has ideas.

**Comment by plexish** [comment]
DigitalOcean droplet going up!
